tests/testthat/test-use-roxygen.R

```diff
@@ -11,19 +11,3 @@ test_that("use_roxygen_md() adds DESCRIPTION fields to naive package", {
   expect_true(desc::desc_has_fields("RoxygenNote", pkg))
   expect_true(uses_roxygen_md())
 })
-
-test_that("use_roxygen_md() does not error on a roxygen-using package", {
-  skip_if_not_installed("roxygen2")
-  with_mock(
-    ## need to pass the check re: whether roxygen2md is installed
-    `usethis:::check_installed` = function(pkg) TRUE, {
-      scoped_temporary_package()
-      cat(
-        "RoxygenNote: 6.0.1\n",
-        file = proj_path("DESCRIPTION"),
-        append = TRUE
-      )
-      expect_error_free(use_roxygen_md())
-    }
-  )
-})
```

Comment:
Is there a reason to delete this / delete this in this PR?

Reply:
Related: should we add some tests around the new features?

Reply:
It didn't really apply any more. I'll have a play with some tests to see if they feel worthwhile.

Reply:
I had try, but they're mostly going to be golden tests, so I don't think they're particularly important for this change.